default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, makeWrapper
+, buildkit
+}:
+
+buildGoModule rec {
+  pname = "bass";
+  version = "0.0.1-alpha";
+  src = ./.;
+
+  # get using ./hack/get-nix-vendorsha
+  vendorSha256 = "sha256-YauCD97LRd649ag/EQ8VrNc90oXszk/brBcj33mvjrM=";
```

Comment:
When do I need to run this `./hack/` script? When a Go dependency changes?

Reply:
Pretty much changes made to the go.mod & go.sum files

Reply:
Gotcha - I'm definitely gonna forget to do that at least once. Is there a way to catch that sort of thing in CI? I tried changing `go.mod` + `go.sum` and re-ran `nix flake check` and it succeeded. 🤔